### DISCUSSION TITLE
Feasibility of xpore to work on non-nanopore RNA-seq data

### DISCUSSION THREAD

**Discussion by ybfu88** [GoekeLab/xpore#95] [Q&A] [answered]
Dear Dr. GÖKE:

I read with great interest your xpore application published in Nat Biotech to evaluate differential RNA modification using nanopore RNA-seq data. I am wondering if we can use xpore to analyze the other RNA-seq data generated by other sequencer platform such as HiSeq 2000 by converting the fastq (acquired from other sequencing platforms) to fast5 files (via other software such as fast5seek;https://github.com/mbhall88/fast5seek)? Or do you have some software we can apply (rather than fast5seek) to do the conversion from fastq to fast5 files for xpore application? Thank you for your support.

Yong-Bi Fu

**Comment by jonathangoeke** [comment, marked as answer]
Dear Yong-Bi Fu,

Thanks for your interest in our work! xPore requires data that was processed by [Nanopolish](https://github.com/jts/nanopolish) as input, so anything that works with Nanopolish would work with xPore. I am not aware of a software that translates fastq into fast5 for such a purpose, so in practice I believe xPore might currently only work with Nanopore direct RNA-Seq data.

Thanks
Jonathan
